### PR TITLE
Preset name backend

### PR DESCRIPTION
**Pull request by egedebruin**
I have the name to the preset objects. The database is also updated and the endpoint also add a name to the preset, as well als editing a name.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/preset/Preset.java [hunk 1]
```diff
@@ -130,6 +141,9 @@ public boolean equals(Object o) {
     if (!tags.equals(preset.getTags())) {
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/preset/Preset.java]: I think these statements can be combined with an `|`

**Comment by jochem725** [comment]
Great to see this added. I agree with merging.

File: backend/src/main/java/com/benine/backend/camera/PresetCamera.java [hunk 2]
```diff
@@ -11,10 +11,11 @@
   
   /**
    * Creates a preset from a camera.
-   * @param tagList   The tag belonging to the preset. 
+   * @param tagList   The tag belonging to the preset.
+   * @param name      The name of the name
```
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/camera/PresetCamera.java]: I think you mean the name of the preset

File: backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java [hunk 3]
```diff
@@ -46,14 +46,15 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
       
       Camera camera = getCameraController().getCameraById(Integer.parseInt(camID));
       String tags = request.getParameter("tags");
+      String name = request.getParameter("name");
```
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java]: Maybe you could test here if the name is not null or do we allow a preset without a name?

File: backend/src/main/java/com/benine/backend/http/EditPresetHandler.java [hunk 4]
```diff
@@ -43,23 +43,27 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
...
         tagList = new HashSet<>(Arrays.asList(tags.split("\\s*,\\s*"))); 
       }
       if (overwriteTag.equals("true")) {
         updateTag(preset, tagList);
       }
-      
       if (overwritePosition.equals("true")) {
         updatePosition(preset);
       }
+      if (overwriteName.equals("true")) {
```
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/http/EditPresetHandler.java]: I  don't think we need the overwrite parameter, you could make simply this distinction by checking if the name parameter is null or not.

**Comment by NaomidR** [comment]
When you have fixed Bryans comments I agree with merging this.

**Comment by egedebruin** [comment]
Thank you for the comments @bvanwijk , I think I have fixed them now.

**Comment by bvanwijk** [comment]
Looks great, now I agree with merging.

**Pull request merged**